frontend/package.json

```diff
@@ -23,6 +23,7 @@
     "lodash.reduce": "^4.6.0",
     "lodash.sortby": "^4.7.0",
     "moment": "^2.22.2",
+    "node-fetch": "^2.3.0",
```

Comment:
Don't we already have a library for performing HTTP requests?

Reply:
> Don't we already have a library for performing HTTP requests?

Right, `axios`… forgot about that one 🙂